gallia-prehistoire.csl

```diff
@@ -1,17 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" delimiter-precedes-et-al="never" initialize-with="." page-range-format="expanded" demote-non-dropping-particle="sort-only" default-locale="fr-FR">
-  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
   <info>
-    <title>Gallia Préhistoire (Français)</title>
+    <title>Gallia Préhistoire (French)</title>
```

Comment:
please revert this -- we're now using all language labels in the actual language